.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -6,6 +6,3 @@ contact_links:
   - name: '📖 SDK documentation'
     url: https://kinde.com/docs
     about: View the documentation for all of the Kinde SDKs.
-  - name: '🚨 Security vulnerability'
-    url: https://kinde.com/docs/important-information/vulnerability-disclosure-policy
-    about: Privately report a security vulnerability.
```

Comment:
This ↑ gets auto-generated in the issue template chooser when you have a `SECURITY.MD` health file.